src/sentry/api/serializers/models/incident.py

```diff
@@ -0,0 +1,43 @@
+from __future__ import absolute_import
+
+from collections import defaultdict
+
+import six
+
+from sentry.api.serializers import (
+    Serializer,
+    register,
+)
+from sentry.incidents.models import (
+    Incident,
+    IncidentProject,
+)
+
+
+@register(Incident)
+class IncidentSerializer(Serializer):
+    def get_attrs(self, item_list, user, **kwargs):
+        incident_projects = defaultdict(list)
+        for incident_project in IncidentProject.objects.filter(
+                incident__in=item_list).select_related('project'):
+            incident_projects[incident_project.incident_id].append(incident_project.project.slug)
+
+        results = {}
+        for item in item_list:
+            results[item] = {'projects': incident_projects.get(item.id, [])}
+        return results
+
+    def serialize(self, obj, attrs, user):
+        return {
+            'id': six.text_type(obj.id),
+            'identifier': obj.identifier,
```

Comment:
Is this a URL safe slug?

Reply:
Just an int. We couldn't think of a great name, but it represents the number you see here: https://www.figma.com/proto/I7oNk99EkE4GHbwGChCDUsJw/Incidents?node-id=322%3A1&scaling=min-zoom&redirected=1 (Incidents > #2381)